packages/lib/models/settings/builtInMetadata.ts

```diff
@@ -1660,6 +1660,15 @@ const builtInMetadata = (Setting: typeof SettingType) => {
 			isGlobal: true,
 		},
 
+		'cli.folderCollapseState': {
```

Comment:
We already have the setting `collapsedFolderIds`, please use this. Maybe you can also reuse some of the existing logic?

Reply:
Agree for reusing the logic.
As for using the same setting this would mean that the setting would be common across the desktop and cli right?
Is this what users would expect?
I don't have a strong opinion,  most of the time I've everything collapsed anyway.